docs/devices/ROB_200-024-0.md

```diff
@@ -0,0 +1,6 @@
+<!-- Notes BEGIN -->
+## Notes
+All buttons working, only on/off-functioning tested (other functionality like dimming/brightness not tested) 
+Button trigger reference, main on/off = 'on', 'off', buttons numbered 1 to 4 = 'on_1', 'off_1, 'on_2', 'off_2, etc., button S1 = 'recall_1_1', button S2 = 'recall_2_1'
+Source based on Zigbee model 'ZGRC-KEY-013', model: 'SR-ZG9001K12-DIM-Z4', vendor: 'Sunricher'
```

Comment:
This info is not needed since it will be generated automatically.

Reply:
Removed the .md file.